### ISSUE TITLE
AQAvit Meeting July 8, 2020

### ISSUE THREAD

**Issue by smlambert** [AdoptOpenJDK/openjdk-tests#1869]
AQAvit Meeting July 8, 2020
Wednesday, July 8th 11:30am – 12:00pm (GMT-04:00)
Description: Bi-weekly meeting to discuss AdoptOpenJDK Quality Assurance (AQA) topics.

Join with Google Meet
meet.google.com/ejp-yjon-vec

Join by phone
PIN: ‪754 815 097‬#

Proposed Agenda:

Alternate sources for test dependencies (discuss/decide best way to accomplish these 2):
- Test dependencies from alternate sources https://github.com/AdoptOpenJDK/openjdk-tests/issues/1841 - @d3r3kk 
- Override docker image name - https://github.com/AdoptOpenJDK/openjdk-tests/pull/1833 - @jennli

Smart parallelization enhancements:
- Cache TRSS JSON output https://github.com/AdoptOpenJDK/openjdk-tests/pull/1865 and https://github.com/AdoptOpenJDK/TKG/pull/91 - thanks to @llxia @renfeiw 

Notable Misc:
- Enable jdk_awt/jdk_swing test targets on AIX (now able to set DISPLAY https://github.com/AdoptOpenJDK/openjdk-tests/pull/1835, thanks to @Haroon-Khel) 
- New bumblebench test target https://github.com/AdoptOpenJDK/openjdk-tests/pull/1862 - thanks to @nikolamilijevic1 
- Rerun link auto-populated https://github.com/AdoptOpenJDK/openjdk-tests/pull/1850 - thanks to @llxia 
- java version extraction https://github.com/AdoptOpenJDK/openjdk-tests/pull/1840 and https://github.com/AdoptOpenJDK/openjdk-test-tools/pull/285 - thanks to @OscarQQ 
- git diff in "create new issue" TRSS feature https://github.com/AdoptOpenJDK/openjdk-test-tools/pull/290 - thanks to @OscarQQ 

Any other business

**Comment by smlambert** [comment]
Recording [link for July 8th meeting](https://drive.google.com/file/d/17tlLh6_sLrXQMlsITgMV0Z3pVX8qoiv5/view?usp=sharing)

Minutes:
- Derek gave an overview and background motivation for issue #1841 and PR #1833, and described the internal Microsoft environment.  We are all in agreement that creating this easier flexibility is advantageous, and between the 2 options proposed in 1841, there is a slight hybrid which still allows developers to run a single test target locally on their laptop while extending getDependencies make target called by TKG so that it can pick up more than just functional dependencies, but also other ones defined in other test subdirs (perf/external and so forth).  Shelley has asked Jim Crowley on the draft PR to merge Elastic changes, once that lands, we can ask Jennifer to update her PR.  In tandem, the approach to update getDependencies for other test groups can happen, need to add a few more notes to #1841, then create a draft PR to ensure it works for all environments.

- Renfei and Lan gave a brief overview of the final piece of the smart parallelization work (that covers the fallback plan when direct querying of TRSS does not work). 

- quick mention of all the Notable miscellaneous that have been recently merged, and a note from Oscar on some enhancements to the "Create new issue" feature in TRSS

- Sophia asked about caching and noted that the test dependency topic is also relevant to her work in the build-jdk github action.  

- Derek offered to do a quick run through of Azure Devops for anyone who wants one.  He can do a short overview of the public AzDo in next AQAvit meeting, especially once Joseph's [openjdk-build PR](https://github.com/AdoptOpenJDK/openjdk-build/pull/1648) lands.

- checked that this regularly scheduled time works for all attendees (from Pacific coast to Hursley).  It (just barely) works, we will keep this time slot going forward.
